packages/eas-cli/src/metadata/config.ts

```diff
@@ -0,0 +1,42 @@
+import Ajv from 'ajv';
+import assert from 'assert';
+
+import { AppleConfigReader } from './apple/config/reader';
+import { AppleConfigWriter } from './apple/config/writer';
+import { AppleMetadata } from './apple/types';
+
+export interface MetadataConfig {
+  /** The store configuration version */
+  configVersion: number;
+  /** All App Store related configuration */
+  apple?: AppleMetadata;
+}
+
+/**
+ * Run the JSON Schema validation to normalize defaults and flag early config errors.
+ * This includes validating the known store limitations for every configurable property.
+ */
+export function validateConfig(config: unknown): {
+  valid: boolean;
+  errors: Ajv.ErrorObject[];
+} {
+  const validator = new Ajv({ allErrors: true, useDefaults: true })
+    .addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'))
```

Comment:
Is there any reason you are using require instead of import?

Reply:
I don't think this reason applies to EAS CLI, but in the new ES module syntax you can't just import JSON files like `import('path/to/file.json')` anymore. It requires a [new syntax from ES JSON modules](https://www.stefanjudis.com/snippets/how-to-import-json-files-in-es-modules-node-js/), so I try to avoid using `import` for plain JSON files in case we want to move over to ES modules.